perf/bonnie.py

```diff
@@ -36,12 +40,14 @@ class Bonnie(Test):
     """
 
     def setUp(self):
+
         """
         Build bonnie++
         Source:
-         http://www.coker.com.au/bonnie++/experimental/bonnie++-1.96.tgz
+         http://www.coker.com.au/bonnie++/bonnie++-1.03e.tgz
         """
-        tarball = self.fetch_asset('http://www.coker.com.au/bonnie++/experimental/bonnie++-1.96.tgz')
+        tarball = self.fetch_asset('http://www.coker.com.au/\
+                                    bonnie++/bonnie++-1.03e.tgz')
```

Comment:
please use expire param in` fetch_asset` as it help to download again it if expire time got expired .